pom.xml

```diff
@@ -101,7 +101,7 @@
         <dependency>
             <groupId>io.confluent</groupId>
             <artifactId>kafka-avro-serializer</artifactId>
-            <version>2.0.1</version>
+            <version>3.3.1</version>
```

Comment:
Any reason not to use latest?

By the way, the Avro version might need downgraded with this. 
https://github.com/confluentinc/common/commit/8141194741de177071b0f0ab6f6faf2aa5defdd6#diff-600376dffeb79835ede4a0b285078036